draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -922,9 +955,15 @@
         </t>
 
         <t>A network device that monitors the transport layer will track the
-        state of TCP sessions, such as TCP sequence numbers.  TCP
-        encapsulation of IKE should therefore use common TCP behaviors to
-        avoid being dropped by middleboxes.
+        state of TCP sessions, such as TCP sequence numbers.  If the IKE implementation 
+        has its own minimal implementation of TCP,
+        it SHOULD still use common TCP behaviors to avoid being dropped by
+        middleboxes.
+        </t>
+	      
+        <t>Operators that intentionally block IPsec because of security implications 
+        might want to also block TCP port 4500 or use other methods to reject TCP encapsulated IPsec traffic 
+	(e.g. filter out TCP connections that begin with the "IKETCP" stream prefix).
```

Comment:
I slightly modified this sentence to make it more generic. Is it OK?

Reply:
Looks good!